ChangeLog.md

```diff
@@ -8,6 +8,10 @@
 * Remove UTF-8 BOM from a few files
 * [plugin-template.c](src/plugin-template.c): add state impl and some comments
 * [audio-ports-activation.h](include/clap/ext/draft/audio-ports-activation.h): improved documentation
+* [version.h](include/clap/version.h): Add a CLAP_VERSION_GE(maj,min,rev), _EQ and _LT macro. Remove the
```

Comment:
Maybe use the back quotes for the programming names?

Reply:
Sure just did that and squashed down all my commits and force pushed.